cfgmgr/teammgr.cpp

```diff
@@ -112,18 +112,53 @@ void TeamMgr::doTask(Consumer &consumer)
     }
 }
 
-
 void TeamMgr::cleanTeamProcesses()
 {
     SWSS_LOG_ENTER();
     SWSS_LOG_NOTICE("Cleaning up LAGs during shutdown...");
-    for (const auto& it: m_lagList)
+
+    std::unordered_map<std::string, pid_t> aliasPidMap;
+
+    for (const auto& alias: m_lagList)
+    {
+        std::string res;
+        pid_t pid;
+
+        {
+            std::stringstream cmd;
+            cmd << "cat " << shellquote("/var/run/teamd/" + alias + ".pid");
+            EXEC_WITH_ERROR_THROW(cmd.str(), res);
```

Comment:
My suggestion would be to replace all calls to EXEC_WITH_ERROR_THROW here with API's like kill(). This is because using this also execs a new process each time and would be a time consumer if we have large number of portchannels.

We had the implementation with kill(pid, signo) earlier which was the best I would tell.

Reply:
@judyjoseph done